patches/node.v18.2.0.cpp.patch

```diff
@@ -344,15 +344,15 @@ index 0000000000..fb2d47f52b
      CHECK(args[2]->IsNumber());
      line_offset = args[2].As<Int32>()->Value();
      CHECK(args[3]->IsNumber());
-@@ -732,6 +734,7 @@ void ContextifyScript::New(const FunctionCallbackInfo<Value>& args) {
+@@ -724,6 +726,7 @@ void ContextifyScript::New(const FunctionCallbackInfo<Value>& args) {
        CHECK_NOT_NULL(sandbox);
        parsing_context = sandbox->context();
      }
 +    sourceless = args[7]->IsTrue();
```

Comment:
I'm really having a hard time understanding how this is included, I've taken the patch as is from main, renamed it, and only changed the hunk line numbers, so this additional line baffles me

Reply:
oh I see, the github highlighting is getting confused. this line exists in the original file, but since it has the + at the beginning github highlights it as if it were being added in this patch.